pyproject.toml

```
@@ -31,9 +31,25 @@ classifiers = [
     "Programming Language :: Python :: 3",
 ]
 requires-python = ">=3.8"
```

Let's update this while we're at it. RAPIDS requires Python 3.9.
```suggestion
requires-python = ">=3.9"
```